launch/ardrone.launch

```diff
@@ -1,22 +1,21 @@
 <launch>
-<arg name="ns"/>
-<arg name="mavlink_fcu_url" default="udp://localhost:14565@localhost:14560"/>
+    <arg name="ns"/>
```

Comment:
I'm pretty sure we can delete this one. Those parameters don't even exist anymore.